WotC material/v13/pub_20161115_VGtM.js

```diff
@@ -503,13 +503,14 @@ RaceList["triton"] = {
 	},
 	languageProfs : ["Common", "Primordial"],
 	dmgres : ["Cold"],
+	vision : [["Darkvision", 60]],
 	age : " reach maturity around age 15 and can live up to 200 years",
 	height : " are around 5 feet tall (4'6\" + 2d10\")",
 	weight : " weigh around 150 lb (90 + 2d10 \xD7 2d4 lb)",
 	heightMetric : " are around 1,6 metres tall (135 + 5d10 cm)",
 	weightMetric : " weigh around 70 kg (40 + 5d10 \xD7 4d4 / 10 kg)",
 	scores : [1, 0, 1, 0, 0, 1],
-	trait : "Triton (+1 Strength, +1 Constitution +1 Charisma)\nControl Air and Water: I can cast the Fog Cloud spell. Once I reach 3rd level, I can cast the Gust of Wind spell. Once I reach 5th level, I can cast the Wall of Water spell. All three spells can be used once per long rest. Charisma is my spellcasting ability for these spells.\nEmissary of the Sea: I can communicate simple ideas to beasts that can breathe water.\nGuardians of the Depths: I have resistance to cold damage and ignore all drawbacks from a deep, underwater environment." + (typePF ? "\n" : " ") + "Amphibious: I can breathe air and water.",
+	trait : "Triton (+1 Strength, +1 Constitution +1 Charisma)\nControl Air and Water: I can cast the Fog Cloud spell. Once I reach 3rd level, I can cast the Gust of Wind spell. Once I reach 5th level, I can cast the Wall of Water spell. All three spells can be used once per long rest. Charisma is my spellcasting ability for these spells.\nEmissary of the Sea: I can communicate simple ideas to beasts that can breathe water.\nGuardians of the Depths: Adapted to even the most extreme ocean depths, I have resistance to cold damage." + (typePF ? "\n" : " ") + "Amphibious: I can breathe air and water.",

```

Comment:
Not sure why this was changed, though. Any particular reason? Looks like only Darkvision was added to Triton.

Reply:
**Guardians of the Depths (p. 118).** This trait has been changed to read as follows: “Adapted to even the most extreme ocean depths, you have resistance to cold damage.” Is the word for word description of the trait, from the errata, accounting for first person perspective.

The errata places a lot less emphasis on the "drawbacks from a deep, underwater environment" and seems more focused on just justifying the cold resistance.

Reply:
Ah right you are, sorry, couldn't see the wood for the trees! :+1: